src/Frame.js

```diff
@@ -0,0 +1,49 @@
+function Frame(finalFrame) {
```

Comment:
The name for this parameter should make it obvious it expects a boolean. Something like `isFinalFrame` would be nice.